### PR TITLE
Fix pypy environment in Travis.

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

**Comment by pferate** [comment]
I guess I'm a bit slow here.  I filed #596, which this will resolve.

**Comment by pferate** [comment]
@jonparrott, with `pypy-2.6.0` already installed, `pyenv` is prompting for confirmation on installation.

File: scripts/install.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ fi
 
 # Travis ships with an old version of PyPy, so install at least version 2.6.
 if [[ "${TOX_ENV}" == "pypy" ]]; then
-    git clone https://github.com/yyuu/pyenv.git ${HOME}/.pyenv
+    if [ ! -d "${HOME}/.pyenv" ]; then
+        git clone https://github.com/yyuu/pyenv.git ${HOME}/.pyenv
+    fi
     ${HOME}/.pyenv/bin/pyenv install pypy-2.6.0
```
Comment (pferate) [inline scripts/install.sh]: Probably want to use `--skip-existing` (or maybe `--force`) for the `install`

**Comment by jonparrott** [comment]
Thanks for filing the bug. It'll feel good to close it. :P

File: scripts/install.sh (hunk 1 above)
Comment (theacodes) [inline scripts/install.sh]: Good idea.

**Comment by jonparrott** [comment]
Alright tests for the test god are passing, @nathanielmanistaatgoogle ready for you to review/merge.

**Pull request merged**